docs/components/modeler/desktop-modeler/troubleshooting.md

```diff
@@ -0,0 +1,100 @@
+---
+id: troubleshooting
+title: Fix common issues with the Desktop Modeler
+description: "This page lists common issues with the Desktop Modeler and potential resolutions."
+---
+
+## How to start the Modeler without plug-ins?
+
+You can start the modeler with the [`--disable-plugins` flag](https://docs.camunda.io/docs/components/modeler/desktop-modeler/flags/#disable-plug-ins).
+
+## How to obtain the Modeler logs?
+
+Depending on your operating system you can find the Modeler logs in different places:
+
+#### Windows
+
+```plain
+%APPDATA%\Camunda Modeler\logs
+```
+
+#### Mac OS
+
+```plain
+~/Library/Logs/Camunda Modeler
+```
+
+#### Linux
+
+```plain
+~/.config/camunda-modeler/logs
+```
+
+## I cannot connect to Zeebe
+
+> I try to connect to deploy to a remote Zeebe instance and the modeler tells me "cannot find a running Zeebe".
```

Comment:
Hmm so this enforces a structure where there is always a linkable heading and a quotation right below it. It's OK this way. I just wanted to avoid a situation where a general heading links to multiple answers.

Reply:
Let's discuss this. The goal is not to be an FAQ but a troubleshooting document. And each section / situation the user is in may have a number of resolutions (which we do not know in advance). In fact this document should help the user to resolve their issues unattended, i.e. without our help.

Reply:
@barmac I tried to simplify the structure/formatting to make the page easier to digest. As you requested this also includes adding additional headers (that we can link to).

I'm still not a fan of these; due to the nature of lengthy headers these deep links are prone to break.